app/assets/javascripts/global.js.coffee

```diff
@@ -47,15 +47,16 @@ $ ->
     $(this).blur()
 
   $(document).on 'click', 'button.notes.list',  ->
```

Comment:
Instead of the `unless` you could add `:not(.disabled)` to the selector `button.notes.list:not(.disabled)` but that's optional